### DISCUSSION TITLE
Would it be possible to export all types from eip1193.ts?

### DISCUSSION THREAD

**Discussion by jerryjfwang** [wevm/viem#568] [Idea / Feature Request]
I'm trying to implement the EIP1193 standard, and it'd be really helpful to be able to access types like Events. Thanks!

**Comment by tmm** [comment]
They should be already.

- https://github.com/wagmi-dev/viem/blob/main/src/index.ts#L443-L457

Let us know if there are any you are missing or feel free to open a PR.
